draft/spec/index.html

```diff
@@ -834,6 +834,14 @@ <h2>Version Inventory and Inventory Digest</h2>
             most recent version. Every inventory file MUST have a corresponding <a href="#inventory-digest">Inventory
             Digest</a>.
         </p>
+        <p>
+            In the case that prior version directories include an inventory file there will be multiple inventory 
+            files describing prior versions within the OCFL Object. Each version block in each prior inventory file
+            MUST represent the same object state as the corresponding version block in the current inventory file. 
+            Additionally, the values of the <code>created</code>, <code>message</code> and <code>user</code> keys in
+            each version block in each prior inventory file SHOULD have the same values as the corresponding keys in
+            the correspodings version block in the current inventory file.
```

Comment:
Is this saying that it is possible to change the responsibility of previous versions?

Reply:
typo in 'corresponding' L843

Reply:
Thanks, typo fixed.

This doesn't talk about change, just the state of different version inventories at any point in time. Having said that, I think of it as a rule going forward in time -- I SHOULD NOT change the v1 message in the v2 inventory

Reply:
I guess I'm asking why we're saying we "SHOULD NOT" change the message in v1 when it's referenced in the v2 inventory, instead of "MUST NOT". I would think that all versions of the v1 block in all inventories MUST be exactly the same.

Reply:
Well, that goes back to the question of #421 -- I prefer SHOULD because I can see legitimate reasons for an update yet one would want to ever rewrite the old version inventory

Reply:
What legitimate reasons are there for updating the `created`, `message`, and `user` values of previous versions?

Reply:
Will move discussion back up to #421...